packages/@webex/plugin-meetings/package.json

```diff
@@ -74,6 +74,7 @@
     "@webex/media-helpers": "workspace:*",
     "@webex/plugin-people": "workspace:*",
     "@webex/plugin-rooms": "workspace:*",
+    "@webex/web-capabilities": "1.3.0",
```

Comment:
@brycetham I saw that @webex/internal-media-core is used without `^`. So I thought that we do not use `^` to ensure maximum stability and predictability. Do you know is that so?

Reply:
For internal-media-core I think we took `^` away because we were changing it constantly and we were worried that we might make a breaking change that would affect the SDK. That's less the case for web-capabilities, so I think we should keep the `^` here. (To be honest I think we should think about putting `^` back into internal-media-core as well, but eventually internal-media-core is going to go away so we can just handle it then.)

Reply:
@brycetham It seems like a good idea to stick to this approach, too. Thank you for your comment.